src/components/process-list/process-list.config.yml

```diff
@@ -0,0 +1,6 @@
+label: Process List
+status: wip
+preview: "@uswds-framed"
+context:
+  lists:
+    content: 'Lorem ipsum dolor sit amet, <a href="javascript:void(0);">consectetur adipiscing</a> elit, sed do eiusmod.'
```

Comment:
Can probably get rid of some of this yml here if it's not being used.

Reply:
Addressed, stripped it down to 
label: Process List
status: wip